emcc.py

```diff
@@ -4199,12 +4200,14 @@ def generate_traditional_runtime_html(target, options, js_target, target_basenam
     script.src = None
     script.inline = js_contents
 
-  html_contents = do_replace(shell, '{{{ SCRIPT }}}', script.replacement())
-  html_contents = tools.line_endings.convert_line_endings(html_contents, '\n', options.output_eol)
+  shell = do_replace(shell, '{{{ SCRIPT }}}', script.replacement())
+  shell = shell.replace('{{{ SHELL_CSS }}}', utils.read_file(utils.path_from_root('src/shell.css')))
+  shell = shell.replace('{{{ LOGO_SVG }}}', utils.read_file(utils.path_from_root('src/logo.svg')))
```

Comment:
Maybe `SHELL_LOGO`? The file format isn't that important I think.

Reply:
Done.

Reply:
Maybe not pushed? I still see `LOGO_SVG` and not `SHELL_LOGO`.

Reply:
Done now.  I also made the .svg loadable in github/browser and put in media/ with the other svg files.